pkgs/top-level/python-packages.nix

```
@@ -5892,6 +5892,8 @@ in {
 
   pyviz-comms = callPackage ../development/python-modules/pyviz-comms { };
 
+  pyvips = callPackage ../development/python-modules/pyvips { };
```

to avoid pulling python packages of the same name
```suggestion
  pyvips = callPackage ../development/python-modules/pyvips {
    inherit (pkgs) pkg-config vips glib;
  };
```